include/structs/str_dungeon.h

```diff
@@ -146,11 +146,19 @@ typedef struct FloorProperties
     u8 unk1A;
 } FloorProperties;
 
-typedef struct UnkDungeonGlobal_unk1C590
+enum {
+    ITEM_SPAWN_NORMAL,
+    ITEM_SPAWN_IN_SHOP,
+    ITEM_SPAWN_IN_MONSTER_HOUSE,
+    ITEM_SPAWN_WALL, // ?
```

Comment:
Is there confusion about this? Items can spawn inside walls sometimes.

Reply:
Yeah, I had no idea it was possible. I guess I didn't play enough as a ghost type lol

Reply:
Ah okay, if there's no other confusion then let's remove the question mark comment.

Reply:
Removed